python/cudf/cudf/core/dataframe.py

```diff
@@ -5875,33 +5872,36 @@ def _reduce(
                     "skew",
                 )
 
-                if numeric_only is None and op in numeric_ops:
-                    warnings.warn(
-                        f"The default value of numeric_only in DataFrame.{op} "
-                        "is deprecated. In a future version, it will default "
-                        "to False. In addition, specifying "
-                        "'numeric_only=None' is deprecated. Select only valid "
-                        "columns or specify the value of numeric_only to "
-                        "silence this warning.",
-                        FutureWarning,
-                    )
-                    numeric_cols = (
-                        name
+                if op in numeric_ops:
+                    if numeric_only:
+                        numeric_cols = (
+                            name
+                            for name in self._data.names
+                            if is_numeric_dtype(self._data[name])
+                        )
+                        source = self._get_columns_by_label(numeric_cols)
+                        if source.empty:
+                            return Series(index=cudf.StringIndex([]))
```

Comment:
This is repeating previously done work now. In the old code you could have `numeric_only == None` which wouldn't enter the conditional on line 5842 so would need to be filtered here. Now if filtering is needed it is already done in 5842-5850 so this can be removed.

Reply:
Done